.travis.yml

```diff
@@ -12,11 +12,11 @@ matrix:
         - python -m unittest discover test/
 
     - language: python
-      python: 2.7
```

Comment:
I am not sure you can do that. please check with @ydye who may have some python 2.7 code.

Reply:
This is test cases for watchdog, no unit tests from @ydye involved.